src/openzaak/components/documenten/api/filters.py

```diff
@@ -37,3 +43,40 @@ class Meta:
             "startdatum": ["lt", "lte", "gt", "gte"],
             "einddatum": ["lt", "lte", "gt", "gte"],
         }
+
+
+def object_queryset(request):
+    object_value = request.query_params.get("object", "")
+    if "zaken" in object_value:
```

Comment:
that's not a very elegant way to figure this out. Isn't it possible to leverage `vng_api_common.utils.get_viewset_for_path`, and then use `viewset.get_queryset().model._default_manager.all()`?

Reply:
Thanks for the advice. Changed with the use of `get_viewset_for_path`